karma.conf.js

```diff
@@ -34,7 +34,9 @@ module.exports = function (config) {
       // we use the BDD interface for all of the tests that
       // aren't interface-specific.
       'test/browser-fixtures/bdd.fixture.js',
-      'test/unit/*.spec.js'
+      'test/unit/*.spec.js',
+      'test/browser-unit/*.spec.js',
+      'test/browser-reporters/*.spec.js'
     ]),
```

Comment:
We could put the HTML reporter test in the browser unit folder (it's not like we're planning on adding multiple browser reporters in core), but this is more obvious by parallel of `test/reporters`.